AutoClicker/Views/MainWindow.xaml.cs

```diff
@@ -290,6 +315,11 @@ private void OnClickTimerElapsed(object sender, ElapsedEventArgs e)
                     clickTimer.Stop();
                     ResetTitle();
                 }
+
+                var interval = CalculateInterval();
+                if (IsIntervalValid())

```

Comment:
This calls CalculateInterval() twice, so we are checking a new interval?

Maybe there should also be
`IsIntervalValid(Integer interval)`,
which would be called with the interval calculated above

Reply:
This is a good point. I didn't realize that IsIntervalValid calls CalculateInterval. This commit is part of the hotfix for the bug mentioned in a comment. I wouldn't read too deeply into its design yet. I need to run some cleanup.

Reply:
Should be resolved